fetch/api/basic/stream-response.any.js

```diff
@@ -26,4 +26,17 @@ promise_test(function(test) {
         assert_unreached( "Body does not exist in response");
       });
   });
-}, "Stream response's body");
+}, "Stream response's body when content-type is present");
+
+// This test makes sure that the response body is not buffered if no content type is provided.
+promise_test(function(test) {
+  return fetch(RESOURCES_DIR + "trickle.py?ms=300&count=10&notype=true").then(function(resp) {
+    var count = 0;
```

Comment:
Both here and above maybe drop this and just pass 0 to streamBody or make that argument optional even?